nexus_constructor/model/attributes.py

```diff
@@ -50,6 +53,7 @@ class FieldAttribute:
 
     name = attr.ib(type=str)
     values = attr.ib(type=ValueType, cmp=False)
+    parent_node = attr.ib(type=Attributes, default=None)
```

Comment:
Should `__eq__` also check if `parent_node` are same?

Reply:
I don't think it is needed. The attributes are only checked with __eq__ for the same parent_node. For example if dataset dataset_1 has 3 attributes, they will only be compared to each other and not to another set of attributes in say dataset_2. That comparison would not be necessary ever.